README.md

```diff
@@ -100,11 +100,10 @@ Or you can start by using one of our starter templates:
 const { Nuxt, Builder } = require('nuxt')
 
 // Import and set nuxt.js options
-let config = require('./nuxt.config.js')
+const config = require('./nuxt.config.js')
 config.dev = (process.env.NODE_ENV !== 'production')
 
-let nuxt = new Nuxt(config)
-nuxt.ready().catch(console.error)
```

Comment:
Shouldn't we still advise to wait for nuxt ready (ideally)?

Reply:
Yes but it also caused complains. Maybe for 3.x we can show a more complete example with await. Currently nuxt exactly does this and also shows error on console.